actors/miner/src/lib.rs

```diff
@@ -4678,20 +5017,26 @@ fn validate_seal_aggregate_proof(
     proof: &RawBytes,
     sector_count: u64,
     policy: &Policy,
+    interactive: bool,
 ) -> Result<(), ActorError> {
-    if sector_count > policy.max_aggregated_sectors {
+    let (min, max) = match interactive {
+        false => (policy.min_aggregated_sectors, policy.max_aggregated_sectors),
+        true => (policy.min_aggregated_sectors_ni, policy.max_aggregated_sectors_ni),
```

Comment:
```suggestion
        true => (policy.min_aggregated_sectors, policy.max_aggregated_sectors),
        false => (policy.min_aggregated_sectors_ni, policy.max_aggregated_sectors_ni),
```

Reply:
parameter name still doesn't match usage; either change its name or change the usage in here and at callsites